packages/synapse-react-client/src/components/ColumnFilter/ColumnFilter.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+import { useQueryContext } from '../QueryContext/QueryContext'
+import {
+  FacetColumnResultValues,
+  FacetColumnValuesRequest,
+} from '@sage-bionetworks/synapse-types'
+import { useQuery } from '@tanstack/react-query'
+import { Autocomplete, TextField, Box } from '@mui/material'
+import { getCorrespondingSelectedFacet } from '../../utils/functions/queryUtils'
+
+export type FilterProps = {
+  filterColumnName?: string
+  addValueToSelectedFacet?: (
+    facet: { columnName: string },
+    value: string,
+  ) => void
+  removeSelectedFacet?: (facet: { columnName: string }) => void
+}
```

Comment:
`filterColumnName` will always be defined.

Also, remove the other unused props for the functions that are coming from context

```suggestion
export type FilterProps = {
  filterColumnName: string
}
```